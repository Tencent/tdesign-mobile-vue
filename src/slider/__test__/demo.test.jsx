/**
 * 该文件为由脚本 `npm run test:demo` 自动生成，如需修改，执行脚本命令即可。请勿手写直接修改，否则会被覆盖
 */

import { mount } from '@vue/test-utils';
import baseVue from '@/slider/demos/base.vue';
import disableVue from '@/slider/demos/disable.vue';
import markVue from '@/slider/demos/mark.vue';
import mobileVue from '@/slider/demos/mobile.vue';
import rangeVue from '@/slider/demos/range.vue';
import titleVue from '@/slider/demos/title.vue';
import unZeroVue from '@/slider/demos/unZero.vue';
import valueVue from '@/slider/demos/value.vue';

const mapper = {
  baseVue,
  disableVue,
  markVue,
  mobileVue,
  rangeVue,
  titleVue,
  unZeroVue,
  valueVue,
};

describe('Slider', () => {
  Object.keys(mapper).forEach((demoName) => {
    it(`Slider ${demoName} demo works fine`, () => {
      const wrapper = mount(mapper[demoName]);
      expect(wrapper.element).toMatchSnapshot();
    });
  });
});
