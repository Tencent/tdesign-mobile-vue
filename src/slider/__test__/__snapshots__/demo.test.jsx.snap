// Vitest Snapshot v1

exports[`Slider > Slider baseVue demo works fine 1`] = `
<div
  class="wrapper"
  data-v-5350d6f5=""
>
  <div
    class="t-slider"
    data-v-5350d6f5=""
  >
    <!---->
    <div
      class="t-slider__bar t-slider__bar--default"
    >
      <!---->
      <div
        class="t-slider__line t-slider__line--default t-slider__line--single"
        style=""
      >
        <div
          class="t-slider__dot"
        >
          <!---->
          <div
            class="t-slider__dot-slider"
          />
        </div>
      </div>
    </div>
    <!---->
  </div>
</div>
`;

exports[`Slider > Slider capsuleVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="wrapper"
    data-v-a6c0268e=""
  >
    <div
      class="t-slider"
      data-v-a6c0268e=""
    >
      <!---->
      <div
        class="t-slider__bar t-slider__bar--capsule"
      >
        <!---->
        <div
          class="t-slider__line t-slider__line--capsule t-slider__line--single"
          style=""
        >
          <div
            class="t-slider__dot"
          >
            <!---->
            <div
              class="t-slider__dot-slider"
            />
          </div>
        </div>
      </div>
      <!---->
    </div>
  </div>
  <div
    class="wrapper"
    data-v-a6c0268e=""
  >
    <div
      class="t-slider t-slider--range"
      data-v-a6c0268e=""
    >
      <!---->
      <div
        class="t-slider__bar t-slider__bar--capsule"
      >
        <!---->
        <div
          class="t-slider__line t-slider__line--capsule"
          style="left: 0px; right: 0px;"
        >
          <div
            class="t-slider__dot t-slider__dot--left"
          >
            <!---->
            <div
              class="t-slider__dot-slider"
            />
          </div>
          <div
            class="t-slider__dot t-slider__dot--right"
          >
            <!---->
            <div
              class="t-slider__dot-slider"
            />
          </div>
        </div>
      </div>
      <!---->
    </div>
  </div>
  <div
    class="wrapper"
    data-v-a6c0268e=""
  >
    <div
      class="t-slider t-slider--top t-slider--range"
      data-v-a6c0268e=""
    >
      <!---->
      <div
        class="t-slider__bar t-slider__bar--capsule"
      >
        <!---->
        <div
          class="t-slider__line t-slider__line--capsule"
          style="left: 0px; right: 0px;"
        >
          <div
            class="t-slider__dot t-slider__dot--left"
          >
            <div
              class="t-slider__dot-value"
            >
              0
            </div>
            <div
              class="t-slider__dot-slider"
            />
          </div>
          <div
            class="t-slider__dot t-slider__dot--right"
          >
            <div
              class="t-slider__dot-value"
            >
              0
            </div>
            <div
              class="t-slider__dot-slider"
            />
          </div>
        </div>
      </div>
      <!---->
    </div>
  </div>
  <div
    class="wrapper"
    data-v-a6c0268e=""
  >
    <div
      class="t-slider"
      data-v-a6c0268e=""
    >
      <!---->
      <div
        class="t-slider__bar t-slider__bar--capsule"
      >
        <!---->
        <div
          class="t-slider__line t-slider__line--capsule t-slider__line--single"
          style=""
        >
          <div
            class="t-slider__dot"
          >
            <!---->
            <div
              class="t-slider__dot-slider"
            />
          </div>
        </div>
      </div>
      <!---->
    </div>
  </div>
  <div
    class="wrapper"
    data-v-a6c0268e=""
  >
    <div
      class="t-slider t-slider--range"
      data-v-a6c0268e=""
    >
      <!---->
      <div
        class="t-slider__bar t-slider__bar--capsule"
      >
        <!---->
        <div
          class="t-slider__line t-slider__line--capsule"
          style="left: 0px; right: 0px;"
        >
          <div
            class="t-slider__dot t-slider__dot--left"
          >
            <!---->
            <div
              class="t-slider__dot-slider"
            />
          </div>
          <div
            class="t-slider__dot t-slider__dot--right"
          >
            <!---->
            <div
              class="t-slider__dot-slider"
            />
          </div>
        </div>
      </div>
      <!---->
    </div>
  </div>
  
</div>
`;

exports[`Slider > Slider disabledVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="wrapper"
    data-v-53635720=""
  >
    <div
      class="t-slider t-slider--disabled"
      data-v-53635720=""
    >
      <!---->
      <div
        class="t-slider__bar t-slider__bar--default t-slider__bar--disabled"
      >
        <!---->
        <div
          class="t-slider__line t-slider__line--default t-slider__line--single t-slider__line--disabled"
          style=""
        >
          <div
            class="t-slider__dot"
          >
            <!---->
            <div
              class="t-slider__dot-slider"
            />
          </div>
        </div>
      </div>
      <!---->
    </div>
  </div>
  <div
    class="wrapper"
    data-v-53635720=""
  >
    <div
      class="t-slider t-slider--top t-slider--disabled t-slider--range"
      data-v-53635720=""
    >
      <text
        class="t-slider__range-extreme t-slider__range-extreme--min"
      >
        0
      </text>
      <div
        class="t-slider__bar t-slider__bar--default t-slider__bar--disabled"
      >
        <!---->
        <div
          class="t-slider__line t-slider__line--default t-slider__line--disabled"
          style="left: 0px; right: 0px;"
        >
          <div
            class="t-slider__dot t-slider__dot--left"
          >
            <div
              class="t-slider__dot-value"
            >
              0
            </div>
            <div
              class="t-slider__dot-slider"
            />
          </div>
          <div
            class="t-slider__dot t-slider__dot--right"
          >
            <div
              class="t-slider__dot-value"
            >
              0
            </div>
            <div
              class="t-slider__dot-slider"
            />
          </div>
        </div>
      </div>
      <text
        class="t-slider__value t-slider__value--max"
      >
        100
      </text>
    </div>
  </div>
  <div
    class="wrapper"
    data-v-53635720=""
  >
    <div
      class="t-slider t-slider--disabled t-slider--range"
      data-v-53635720=""
    >
      <!---->
      <div
        class="t-slider__bar t-slider__bar--default t-slider__bar--disabled"
      >
        <!---->
        <div
          class="t-slider__line t-slider__line--default t-slider__line--disabled"
          style="left: 0px; right: 0px;"
        >
          <div
            class="t-slider__dot t-slider__dot--left"
          >
            <!---->
            <div
              class="t-slider__dot-slider"
            />
          </div>
          <div
            class="t-slider__dot t-slider__dot--right"
          >
            <!---->
            <div
              class="t-slider__dot-slider"
            />
          </div>
        </div>
      </div>
      <!---->
    </div>
  </div>
  
</div>
`;

exports[`Slider > Slider labelVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="wrapper"
    data-v-61756b00=""
  >
    <div
      class="t-slider t-slider--top"
      data-v-61756b00=""
    >
      <!---->
      <div
        class="t-slider__bar t-slider__bar--default"
      >
        <!---->
        <div
          class="t-slider__line t-slider__line--default t-slider__line--single"
          style=""
        >
          <div
            class="t-slider__dot"
          >
            <div
              class="t-slider__dot-value"
            >
              10
            </div>
            <div
              class="t-slider__dot-slider"
            />
          </div>
        </div>
      </div>
      <!---->
    </div>
  </div>
  <div
    class="wrapper"
    data-v-61756b00=""
  >
    <div
      class="t-slider t-slider--top t-slider--range"
      data-v-61756b00=""
    >
      <text
        class="t-slider__range-extreme t-slider__range-extreme--min"
      >
        0
      </text>
      <div
        class="t-slider__bar t-slider__bar--default"
      >
        <!---->
        <div
          class="t-slider__line t-slider__line--default"
          style="left: 0px; right: 0px;"
        >
          <div
            class="t-slider__dot t-slider__dot--left"
          >
            <div
              class="t-slider__dot-value"
            >
              0
            </div>
            <div
              class="t-slider__dot-slider"
            />
          </div>
          <div
            class="t-slider__dot t-slider__dot--right"
          >
            <div
              class="t-slider__dot-value"
            >
              0
            </div>
            <div
              class="t-slider__dot-slider"
            />
          </div>
        </div>
      </div>
      <text
        class="t-slider__value t-slider__value--max"
      >
        100
      </text>
    </div>
  </div>
  
</div>
`;

exports[`Slider > Slider mobileVue demo works fine 1`] = `
<div
  class="tdesign-mobile-demo"
  data-v-98828574=""
>
  <h1
    class="title"
    data-v-98828574=""
  >
    Slider 滑动选择器
  </h1>
  <p
    class="summary"
    data-v-98828574=""
  >
    用于选择横轴上的数值、区间、档位
  </p>
  <div
    class="tdesign-mobile-demo-block"
    data-v-98828574=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <h2
        class="tdesign-mobile-demo-block__title"
      >
        01 类型
      </h2>
      <p
        class="tdesign-mobile-demo-block__summary"
      >
        单游标滑块
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot"
    >
      
      <div
        class="wrapper"
        data-v-5350d6f5=""
        data-v-98828574=""
      >
        <div
          class="t-slider"
          data-v-5350d6f5=""
        >
          <!---->
          <div
            class="t-slider__bar t-slider__bar--default"
          >
            <!---->
            <div
              class="t-slider__line t-slider__line--default t-slider__line--single"
              style=""
            >
              <div
                class="t-slider__dot"
              >
                <!---->
                <div
                  class="t-slider__dot-slider"
                />
              </div>
            </div>
          </div>
          <!---->
        </div>
      </div>
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block tdesign-mobile-demo-block_notitle"
    data-v-98828574=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <!--v-if-->
      <p
        class="tdesign-mobile-demo-block__summary tdesign-mobile-demo-block_subtitle"
      >
        双游标滑块
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot"
    >
      
      <div
        class="wrapper"
        data-v-58e1d8c9=""
        data-v-98828574=""
      >
        <div
          class="t-slider t-slider--range"
          data-v-58e1d8c9=""
        >
          <!---->
          <div
            class="t-slider__bar t-slider__bar--default"
          >
            <!---->
            <div
              class="t-slider__line t-slider__line--default"
              style="left: 0px; right: 0px;"
            >
              <div
                class="t-slider__dot t-slider__dot--left"
              >
                <!---->
                <div
                  class="t-slider__dot-slider"
                />
              </div>
              <div
                class="t-slider__dot t-slider__dot--right"
              >
                <!---->
                <div
                  class="t-slider__dot-slider"
                />
              </div>
            </div>
          </div>
          <!---->
        </div>
      </div>
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block tdesign-mobile-demo-block_notitle"
    data-v-98828574=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <!--v-if-->
      <p
        class="tdesign-mobile-demo-block__summary tdesign-mobile-demo-block_subtitle"
      >
        带数值单游标/双游标滑块
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot"
    >
      
      
      <div
        class="wrapper"
        data-v-61756b00=""
      >
        <div
          class="t-slider t-slider--top"
          data-v-61756b00=""
        >
          <!---->
          <div
            class="t-slider__bar t-slider__bar--default"
          >
            <!---->
            <div
              class="t-slider__line t-slider__line--default t-slider__line--single"
              style=""
            >
              <div
                class="t-slider__dot"
              >
                <div
                  class="t-slider__dot-value"
                >
                  10
                </div>
                <div
                  class="t-slider__dot-slider"
                />
              </div>
            </div>
          </div>
          <!---->
        </div>
      </div>
      <div
        class="wrapper"
        data-v-61756b00=""
      >
        <div
          class="t-slider t-slider--top t-slider--range"
          data-v-61756b00=""
        >
          <text
            class="t-slider__range-extreme t-slider__range-extreme--min"
          >
            0
          </text>
          <div
            class="t-slider__bar t-slider__bar--default"
          >
            <!---->
            <div
              class="t-slider__line t-slider__line--default"
              style="left: 0px; right: 0px;"
            >
              <div
                class="t-slider__dot t-slider__dot--left"
              >
                <div
                  class="t-slider__dot-value"
                >
                  0
                </div>
                <div
                  class="t-slider__dot-slider"
                />
              </div>
              <div
                class="t-slider__dot t-slider__dot--right"
              >
                <div
                  class="t-slider__dot-value"
                >
                  0
                </div>
                <div
                  class="t-slider__dot-slider"
                />
              </div>
            </div>
          </div>
          <text
            class="t-slider__value t-slider__value--max"
          >
            100
          </text>
        </div>
      </div>
      
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block tdesign-mobile-demo-block_notitle"
    data-v-98828574=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <!--v-if-->
      <p
        class="tdesign-mobile-demo-block__summary tdesign-mobile-demo-block_subtitle"
      >
        带刻度单游标/双游标滑块
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot"
    >
      
      
      <div
        class="wrapper"
        data-v-0f126170=""
      >
        <div
          class="t-slider"
          data-v-0f126170=""
        >
          <!---->
          <div
            class="t-slider__bar t-slider__bar--default"
          >
            <!---->
            <div
              class="t-slider__line t-slider__line--default t-slider__line--single"
              style=""
            >
              <div
                class="t-slider__dot"
              >
                <!---->
                <div
                  class="t-slider__dot-slider"
                />
              </div>
            </div>
          </div>
          <!---->
        </div>
      </div>
      <div
        class="wrapper"
        data-v-0f126170=""
      >
        <div
          class="t-slider t-slider--range"
          data-v-0f126170=""
        >
          <!---->
          <div
            class="t-slider__bar t-slider__bar--default"
          >
            <!---->
            <div
              class="t-slider__line t-slider__line--default"
              style="left: 0px; right: 0px;"
            >
              <div
                class="t-slider__dot t-slider__dot--left"
              >
                <!---->
                <div
                  class="t-slider__dot-slider"
                />
              </div>
              <div
                class="t-slider__dot t-slider__dot--right"
              >
                <!---->
                <div
                  class="t-slider__dot-slider"
                />
              </div>
            </div>
          </div>
          <!---->
        </div>
      </div>
      
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block"
    data-v-98828574=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <h2
        class="tdesign-mobile-demo-block__title"
      >
        02 组件状态
      </h2>
      <p
        class="tdesign-mobile-demo-block__summary"
      >
        滑块禁用状态
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot"
    >
      
      
      <div
        class="wrapper"
        data-v-53635720=""
      >
        <div
          class="t-slider t-slider--disabled"
          data-v-53635720=""
        >
          <!---->
          <div
            class="t-slider__bar t-slider__bar--default t-slider__bar--disabled"
          >
            <!---->
            <div
              class="t-slider__line t-slider__line--default t-slider__line--single t-slider__line--disabled"
              style=""
            >
              <div
                class="t-slider__dot"
              >
                <!---->
                <div
                  class="t-slider__dot-slider"
                />
              </div>
            </div>
          </div>
          <!---->
        </div>
      </div>
      <div
        class="wrapper"
        data-v-53635720=""
      >
        <div
          class="t-slider t-slider--top t-slider--disabled t-slider--range"
          data-v-53635720=""
        >
          <text
            class="t-slider__range-extreme t-slider__range-extreme--min"
          >
            0
          </text>
          <div
            class="t-slider__bar t-slider__bar--default t-slider__bar--disabled"
          >
            <!---->
            <div
              class="t-slider__line t-slider__line--default t-slider__line--disabled"
              style="left: 0px; right: 0px;"
            >
              <div
                class="t-slider__dot t-slider__dot--left"
              >
                <div
                  class="t-slider__dot-value"
                >
                  0
                </div>
                <div
                  class="t-slider__dot-slider"
                />
              </div>
              <div
                class="t-slider__dot t-slider__dot--right"
              >
                <div
                  class="t-slider__dot-value"
                >
                  0
                </div>
                <div
                  class="t-slider__dot-slider"
                />
              </div>
            </div>
          </div>
          <text
            class="t-slider__value t-slider__value--max"
          >
            100
          </text>
        </div>
      </div>
      <div
        class="wrapper"
        data-v-53635720=""
      >
        <div
          class="t-slider t-slider--disabled t-slider--range"
          data-v-53635720=""
        >
          <!---->
          <div
            class="t-slider__bar t-slider__bar--default t-slider__bar--disabled"
          >
            <!---->
            <div
              class="t-slider__line t-slider__line--default t-slider__line--disabled"
              style="left: 0px; right: 0px;"
            >
              <div
                class="t-slider__dot t-slider__dot--left"
              >
                <!---->
                <div
                  class="t-slider__dot-slider"
                />
              </div>
              <div
                class="t-slider__dot t-slider__dot--right"
              >
                <!---->
                <div
                  class="t-slider__dot-slider"
                />
              </div>
            </div>
          </div>
          <!---->
        </div>
      </div>
      
      
    </div>
  </div>
  <div
    class="tdesign-mobile-demo-block"
    data-v-98828574=""
  >
    <div
      class="tdesign-mobile-demo-block__header"
    >
      <h2
        class="tdesign-mobile-demo-block__title"
      >
        03 特殊样式
      </h2>
      <p
        class="tdesign-mobile-demo-block__summary"
      >
        胶囊型滑块
      </p>
    </div>
    <div
      class="tdesign-mobile-demo-block__slot"
    >
      
      
      <div
        class="wrapper"
        data-v-a6c0268e=""
      >
        <div
          class="t-slider"
          data-v-a6c0268e=""
        >
          <!---->
          <div
            class="t-slider__bar t-slider__bar--capsule"
          >
            <!---->
            <div
              class="t-slider__line t-slider__line--capsule t-slider__line--single"
              style=""
            >
              <div
                class="t-slider__dot"
              >
                <!---->
                <div
                  class="t-slider__dot-slider"
                />
              </div>
            </div>
          </div>
          <!---->
        </div>
      </div>
      <div
        class="wrapper"
        data-v-a6c0268e=""
      >
        <div
          class="t-slider t-slider--range"
          data-v-a6c0268e=""
        >
          <!---->
          <div
            class="t-slider__bar t-slider__bar--capsule"
          >
            <!---->
            <div
              class="t-slider__line t-slider__line--capsule"
              style="left: 0px; right: 0px;"
            >
              <div
                class="t-slider__dot t-slider__dot--left"
              >
                <!---->
                <div
                  class="t-slider__dot-slider"
                />
              </div>
              <div
                class="t-slider__dot t-slider__dot--right"
              >
                <!---->
                <div
                  class="t-slider__dot-slider"
                />
              </div>
            </div>
          </div>
          <!---->
        </div>
      </div>
      <div
        class="wrapper"
        data-v-a6c0268e=""
      >
        <div
          class="t-slider t-slider--top t-slider--range"
          data-v-a6c0268e=""
        >
          <!---->
          <div
            class="t-slider__bar t-slider__bar--capsule"
          >
            <!---->
            <div
              class="t-slider__line t-slider__line--capsule"
              style="left: 0px; right: 0px;"
            >
              <div
                class="t-slider__dot t-slider__dot--left"
              >
                <div
                  class="t-slider__dot-value"
                >
                  0
                </div>
                <div
                  class="t-slider__dot-slider"
                />
              </div>
              <div
                class="t-slider__dot t-slider__dot--right"
              >
                <div
                  class="t-slider__dot-value"
                >
                  0
                </div>
                <div
                  class="t-slider__dot-slider"
                />
              </div>
            </div>
          </div>
          <!---->
        </div>
      </div>
      <div
        class="wrapper"
        data-v-a6c0268e=""
      >
        <div
          class="t-slider"
          data-v-a6c0268e=""
        >
          <!---->
          <div
            class="t-slider__bar t-slider__bar--capsule"
          >
            <!---->
            <div
              class="t-slider__line t-slider__line--capsule t-slider__line--single"
              style=""
            >
              <div
                class="t-slider__dot"
              >
                <!---->
                <div
                  class="t-slider__dot-slider"
                />
              </div>
            </div>
          </div>
          <!---->
        </div>
      </div>
      <div
        class="wrapper"
        data-v-a6c0268e=""
      >
        <div
          class="t-slider t-slider--range"
          data-v-a6c0268e=""
        >
          <!---->
          <div
            class="t-slider__bar t-slider__bar--capsule"
          >
            <!---->
            <div
              class="t-slider__line t-slider__line--capsule"
              style="left: 0px; right: 0px;"
            >
              <div
                class="t-slider__dot t-slider__dot--left"
              >
                <!---->
                <div
                  class="t-slider__dot-slider"
                />
              </div>
              <div
                class="t-slider__dot t-slider__dot--right"
              >
                <!---->
                <div
                  class="t-slider__dot-slider"
                />
              </div>
            </div>
          </div>
          <!---->
        </div>
      </div>
      
      
    </div>
  </div>
</div>
`;

exports[`Slider > Slider rangeVue demo works fine 1`] = `
<div
  class="wrapper"
  data-v-58e1d8c9=""
>
  <div
    class="t-slider t-slider--range"
    data-v-58e1d8c9=""
  >
    <!---->
    <div
      class="t-slider__bar t-slider__bar--default"
    >
      <!---->
      <div
        class="t-slider__line t-slider__line--default"
        style="left: 0px; right: 0px;"
      >
        <div
          class="t-slider__dot t-slider__dot--left"
        >
          <!---->
          <div
            class="t-slider__dot-slider"
          />
        </div>
        <div
          class="t-slider__dot t-slider__dot--right"
        >
          <!---->
          <div
            class="t-slider__dot-slider"
          />
        </div>
      </div>
    </div>
    <!---->
  </div>
</div>
`;

exports[`Slider > Slider stepVue demo works fine 1`] = `
<div
  data-v-app=""
>
  
  <div
    class="wrapper"
    data-v-0f126170=""
  >
    <div
      class="t-slider"
      data-v-0f126170=""
    >
      <!---->
      <div
        class="t-slider__bar t-slider__bar--default"
      >
        <!---->
        <div
          class="t-slider__line t-slider__line--default t-slider__line--single"
          style=""
        >
          <div
            class="t-slider__dot"
          >
            <!---->
            <div
              class="t-slider__dot-slider"
            />
          </div>
        </div>
      </div>
      <!---->
    </div>
  </div>
  <div
    class="wrapper"
    data-v-0f126170=""
  >
    <div
      class="t-slider t-slider--range"
      data-v-0f126170=""
    >
      <!---->
      <div
        class="t-slider__bar t-slider__bar--default"
      >
        <!---->
        <div
          class="t-slider__line t-slider__line--default"
          style="left: 0px; right: 0px;"
        >
          <div
            class="t-slider__dot t-slider__dot--left"
          >
            <!---->
            <div
              class="t-slider__dot-slider"
            />
          </div>
          <div
            class="t-slider__dot t-slider__dot--right"
          >
            <!---->
            <div
              class="t-slider__dot-slider"
            />
          </div>
        </div>
      </div>
      <!---->
    </div>
  </div>
  
</div>
`;
